config/afd.php

```diff
@@ -13,7 +13,16 @@
 
     'entity_implementations' => [
         //
-    ]
+    ],
+
+    'connectors' => [
+        'time' => [
+            'host' => 'https://www.abzportal.nl/ace/sts-ws/stsPort',
```

Comment:
Mogen we deze vrijgeven? @petermein

Reply:
Oh jawel, gaf hij zelf ook over de telefoon